src/api_server.cc

```diff
@@ -204,6 +210,13 @@ void SendMetadataRequest(std::vector<json::value>&& entries,
   request << boost::network::header("Authorization", auth_header);
   request << boost::network::body(request_body);
   http::client::response response = client.post(request);
+  if (status(response) != 200) {
```

Comment:
I would accept anything in the 2xx range, technically a POST can result in a `201 Created`. Applies to other posts as well. I'm sure it's not our contract with the resource metadata api yet, but just something to be mindful of.

Reply:
Done.